### PR TITLE
feat: Added Header and Footer Component

### PR DESCRIPTION
**Pull request by vishal804**
Hello,

Added Header and Footer Component in this pull Request 

Files to be reviewed 
Header.js
Footer.js
Folder structure

Assets,backend and css files can be ignored  

Thank You

### CODE REVIEW

File: src/component/footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        <div class="flex flex-coloum flex-grow-1 brand">
+          <div class="logo">
+            <Link to="/">Shoplify</Link>
+          </div>
+          <p>You Shop Fast We Ship Faster</p>
+          <div class="links-handle flex">
+            <i class="fab fa-facebook fa-2x"></i>
+            <i class="fab fa-instagram fa-2x"></i>
+            <i class="fab fa-twitter fa-2x"></i>
+            <i class="fab fa-snapchat fa-2x"></i>
```
Comment (Developers-codz) [inline src/component/footer/Footer.js]: convert all the class to className

File: src/component/header/Header.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import "./header.css";
+import { Link } from "react-router-dom";
+const Header = () => {
+  return (
+    <div>
```
Comment (Developers-codz) [inline src/component/header/Header.js]: you can use react fragments `<></>` rather than an extra div

**Review by Developers-codz — APPROVED** [review]
There wasn't much to check in the PR. Everything Looks good to merge.

**Review by aayushjain-code — APPROVED** [review]
Looks good to me you can try to add bit of padding on the sides

File: src/component/footer/Footer.js (hunk 1 above)
Comment (vishal804) [reply src/component/footer/Footer.js]: Done the changes

File: src/component/header/Header.js (hunk 2 above)
Comment (vishal804) [reply src/component/header/Header.js]: Done the changes

**Comment by vishal804** [comment]
> Looks good to me you can try to add bit of padding on the sides

**Comment by vishal804** [comment]
> There wasn't much to check in the PR. Everything Looks good to merge.

Thanks For Review

**Comment by vishal804** [comment]
> Looks good to me you can try to add bit of padding on the sides

Will try to do that Thanks for Review

**Pull request merged**